CONTRIBUTING.md

```diff
@@ -533,9 +533,7 @@ Scala has proven the most viable way to do it, as long as you keep the following
 1. If the underlying Scala code requires an `ExecutionContext`, make the Java API take an `Executor` and use
    `ExecutionContext.fromExecutor(executor)` for conversion.
 
-1. Make use of `scala-java8-compat` conversions, see [GitHub](https://github.com/scala/scala-java8-compat)
-   (eg. `scala.compat.java8.FutureConverters` to translate Futures to `CompletionStage`s).
-   Note that we cannot upgrade to a newer version scala-java8-compat because of binary compatibility issues.
+1. Use `org.apache.pekko.util.FutureConverters` to translate Futures to `CompletionStage`s.
```

Comment:
Can we add `ObjectConverters`/`FunctionConverters` in here as well?

Reply:
done

Reply:
Can we also add vice versa after these statements since the converters go both ways?

Reply:
added the 'vice versa' to each